coalib/output/ConsoleInteraction.py

```diff
@@ -599,12 +623,21 @@ def print_actions(console_printer, section, actions, failed_actions):
                             values for the action. If the user did
                             choose to do nothing, return (None, None).
     """
-    choice = choose_action(console_printer, actions)
-
-    if choice == 0:
-        return None, None
-
-    return get_action_info(section, actions[choice - 1], failed_actions)
+    if section.name == 'cli':
```

Comment:
This is still horribly written.
Cant u just simplify it kind like it was before, with the choice first?